website/public/codesandbox-iframe.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GraphQL Mesh Example</title>
+    <style>
+      html,
+      body {
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        height: 100%;
+        overflow: hidden;
+        background-color: #0d0d0d;
+      }
+      iframe {
+        position: fixed;
+        border: 0;
+        width: 100%;
+        height: 100%;
+      }
+      .error {
+        padding: 1rem;
+        font-family: system-ui, sans-serif;
+      }
+    </style>
+  </head>
+  <body>
+    <noscript>JavaScript is required to load the example.</noscript>
+    <iframe
+      src="about:blank"
+      loading="eager"
+      title="Example"
+      className="w-full h-full"
```

Comment:
size-full

Reply:
Actually, this isn't using Tailwind. I forgot to remove that line.